### ISSUE TITLE
Android Notification large_icon not displaying

### ISSUE THREAD

**Issue by amanthegreatone** [evollu/react-native-fcm#270]
Playing with the example app. Large_icon is not displaying in the notification, instead the icon specified is coming.
If icon is not specified then ic_launcher is displayed in both places which I assume is the default coding.

here is the notification code

```
sendRemote(notif) {
    FCM.presentLocalNotification({
      title: notif.title,
      body: notif.body,
      big_text: notif.body,
      priority: "high",
      click_action: notif.click_action,
      show_in_foreground: true,
      local: true,
      icon: "ic_notification",
      large_icon: "ic_notification_big_transparent"
    });
  }
```

I have white transparent pngs for both icon and large_icon as per this pic.

![screen shot 2017-01-12 at 6 59 14 pm](https://cloud.githubusercontent.com/assets/24354452/21891667/ce84551a-d8f9-11e6-8674-0e828f6f1b8c.png)

In native android code we can display colored images for the large_icon. Why can we not do the same here or am I missing something? I have an android app in production that does so. Also other apps like google calendar et al have colored large_icons for notifications.

What verion of RN and react-native-fcm are you running? **0.39.2**
What device are you using? (e.g iOS9 emulator, Android 6 device)? **Android 6 device**
Is your app running in foreground, background or not running? **Foreground**

**Comment by amanthegreatone** [comment]
Ok so after further troubleshooting I found the issue.

In FIRLocalMessagingHelper line 115 is `String largeIcon = bundle.getString("large-icon");`
but we are passing large_icon in the js function FCM.presentLocalNotification(). I changed the large_icon to 'large-icon' and it is working.

Code that works (colored large icon in android)
```
sendRemote(notif) {
    FCM.presentLocalNotification({
      title: notif.title,
      body: notif.body,
      big_text: notif.body,
      priority: "high",
      click_action: notif.click_action,
      show_in_foreground: true,
      local: true,
      icon: "ic_notification",
      'large-icon': "ic_notification_big"
    });
  }
```

Please fix this in the documentation

**Comment by amanthegreatone** [comment]
#269 is also the same issue. Change in java to use large_icon in js or just use 'large-icon' in js.

**Comment by evollu** [comment]
fixed in 4.0.1
